app/lib/ignore.json

```diff
@@ -2,8 +2,7 @@
   "cf-component-demo",
   "cf-grunt-config",
   "cf-tabs",
-  "cf-tables",
   "cf-demo",
   "cf-colors",
   "cf-core"
-]
\ No newline at end of file
+]
```

Comment:
No, it wasn't rebased. I actually used the github interface to edit the file, and something weird might've happened as a result. Locally, the newline was there, so git said there were no changes, thus it was probably just an oddity.